packages/@aws-cdk/aws-apigateway/lib/integrations/lambda.ts

```diff
@@ -72,5 +73,16 @@ export class LambdaIntegration extends AwsIntegration {
         sourceArn: method.testMethodArn,
       });
     }
+
+    const cfnFunction = this.handler.node.defaultChild as lambda.CfnFunction;
+    let deploymentFingerprint;
+    if (!Token.isUnresolved(cfnFunction.functionName)) {
+      const md5 = crypto.createHash('md5');
+      md5.update(JSON.stringify({ functionName: cfnFunction.functionName }));
+      deploymentFingerprint = md5.digest('hex');
```

Comment:
Why do we need to calculate a hash here? Just just the function name or (`JSON.stringify({ functionName }`) as string, isn't that sufficient?

Reply:
That'll work too, but is it a fingerprint anymore? https://en.wikipedia.org/wiki/Fingerprint_(computing).

If not, what is it? (so we can correctly name it)

Reply:
You are right. Maybe `deploymentToken` or `deploymentKey` or something along those lines